src/services/theme.hooks.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { useState, useContext } from 'react';
+import { ThemeContext } from '../frontend/app/app-context-theme/app-context-theme';
+import { Theme } from '../shared/types';
+import themes from '../shared/themes';
+
+// Provider hook that creates theme object and handles state
+export const useProvideTheme = () => {
+  // eslint-disable-next-line prefer-destructuring
+  const defaultTheme = themes[0];
```

Comment:
Why have this when you could just instantiate the useState with whatever the default theme is?

Reply:
true